lib/beaker/host/unix/exec.rb

```diff
@@ -246,7 +246,7 @@ def environment_variable_string_pair_array env
       else
         val = val.to_s
       end
-      env_array << "#{key.to_s.upcase}=\"#{val}\""
```

Comment:
@joshcooper thanks for looking at this! I'll get a PR with that up this week under this same ticket reference to get this fixed!